tests/datasource/fluent/integration/conftest.py

```diff
@@ -57,7 +57,7 @@ def pandas_sql_data(
     df.to_sql("my_table", con=con)
     pandas_ds = context.sources.add_pandas("my_pandas")
     pandas_ds.read_sql(
-        sql="SELECT * FROM my_table",
+        sql=sqlalchemy.sql.text("SELECT * FROM my_table"),
         con=con,
     )
```

Comment:
This is the part that would be changed in the Fluent Datasource, where the string would no longer be passed in as part of the `sql` parameter, but it would need to be a `sqlalchemy.sql.text()` object. 

@NathanFarmer @alexsherstinsky

Reply:
Makes sense to me! We already support `sqlalchemy.sql.text` per the method signature, so I'm good with this.